web/client/jsapi/MapStore2.js

```diff
@@ -188,14 +188,14 @@ const MapStore2 = {
         }, {
             jsAPIEpic: actionTrigger.epic
         });
-        const initialActions = [...getInitialActions(options), loadVersion];
+        const initialActions = [...getInitialActions(options), loadVersion(options.versionURL)];
```

Comment:
you should use bind on loadVersion here, not call the action directly

Reply:
Sorry, where's my head sometimes?